src/network/networkctl.c

```diff
@@ -1060,6 +1110,33 @@ static int link_status_one(
                         return r;
         }
 
+        if (streq_ptr(info->netdev_kind, "bridge")) {
+                r = table_add_many(table,
+                                   TABLE_EMPTY,
+                                   TABLE_STRING, "Forward Delay:",
+                                   TABLE_UINT32, info->forward_delay,
```

Comment:
Hmm, this has no units. I think we shouldn't format this as a timespan, but we can format this as e.g. `200 ms`. (Or what are the units, centiseconds?).